api/src/main/java/org/openmrs/ConditionVerificationStatus.java

```diff
@@ -10,22 +10,8 @@
 package org.openmrs;
 
 /**
- * The ConditionVerificationStatus is what we've historically called "diagnosis certainty" and what FHIR refers to as
```

Comment:
This should not be anywhere in your commit because of https://github.com/openmrs/openmrs-core/commit/7411f5cce4eba1dbc83a89de699f240d124c5502

Reply:
Okay. I'll remove this